frontend/src/component/common/NewConstraintAccordion/NewConstraintAccordionList/NewConstraintAccordionList.tsx

```diff
@@ -150,16 +151,25 @@ export const NewConstraintAccordionList = forwardRef<
                 <ConstraintsList>
                     {constraints.map((constraint, index) =>
                         addEditStrategy ? (
-                            <EditableConstraintWrapper
-                                key={constraint[constraintId]}
-                                constraint={constraint}
-                                onCancel={onCancel.bind(null, index)}
-                                onDelete={onRemove?.bind(null, index)}
-                                onSave={onSave!.bind(null, index)}
-                                onAutoSave={onAutoSave?.(
-                                    constraint[constraintId],
-                                )}
-                            />
+                            state.get(constraint)?.editing ? (
+                                <EditableConstraintWrapper
```

Comment:
So this is interesting. Previously NewConstraintAccordion handled both, edit and view. Now EditableConstraintWrapper  only handles edit, so I need to manually use ConstraintAccordionView for view.